### PR TITLE
chore/refactor4

### PR DESCRIPTION
**Pull request by samuellye98**
Folder restructuring

Change dev settings button

Change proptypes to interface

### CODE REVIEW

File: src/components/Layout/TextRow.tsx [hunk 1]
```diff
@@ -21,9 +20,14 @@ const styles = StyleSheet.create({
...
 
+interface TextRowProps {
+  label: string;
+  text: string;
+}
+
 const imageSource = require("../../assets/blurredtext.jpeg");
 
-const TextRow = ({ label, text }) => {
+export const TextRow = ({ label, text }: TextRowProps) => {
```
Comment (bennchong) [inline src/components/Layout/TextRow.tsx]: Refer to next comment

File: src/components/Modals/InvalidQRModal.tsx [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
-export const InvalidQRModal = ({ handleCloseModal, showModal }) => {
+interface InvalidQRModalProps {
+  handleCloseModal: Function;
+  showModal: boolean;
+}
+
+export const InvalidQRModal = ({
+  handleCloseModal,
+  showModal
+}: InvalidQRModalProps) => {
```
Comment (bennchong) [inline src/components/Modals/InvalidQRModal.tsx]: Refer to the last comment, the code styling is inconsistent

File: src/components/Preview/PreviewContainer.tsx [hunk 3]
```diff
@@ -1,11 +1,18 @@
...
-export const PreviewContainer = ({ navigation, workpass }) => {
+interface PreviewContainerProps {
+  navigation: any;
+  workpass: any;
+}
+
+export const PreviewContainer = ({
+  navigation,
+  workpass
+}: PreviewContainerProps) => {
```
Comment (bennchong) [inline src/components/Preview/PreviewContainer.tsx]: Here too

File: src/components/profile/ProfileName/index.tsx [hunk 4]
```diff
@@ -1,12 +1,25 @@
...
-export const ProfileName = ({ status, photo, fin, name, isPreview }) => {
+interface ProfileNameProps {
+  status: number;
+  photo: string;
+  fin: string;
+  name: string;
+  isPreview: boolean;
+}
+
+export const ProfileName = ({
```
Comment (bennchong) [inline src/components/profile/ProfileName/index.tsx]: And here

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=31

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=31

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=31

**Pull request merged**